.github/label-pr-config.yml

```diff
@@ -0,0 +1,83 @@
+autolabeler:
+  - label: 'Type: documentation'
+    files:
+      - '*.md'
+    # Checks there is no word character (alphanumeric & underscore) before target words, or after to avoid accidental matches
+    title:
+      - '/(?<![\w])((doc)|(docs)|(documentation))(?![\w])/i'
+  - label: 'Type: fix'
```

Comment:
should this be `Type: bugfix` so it will show under the Bug fixes title in release notes?

Reply:
Yes, thank you! Updated to apply the correct label